snap/info_test.go

```diff
@@ -604,7 +604,6 @@ func (s *infoSuite) testDirAndFileMethods(c *C, info snap.PlaceInfo) {
 	c.Check(info.HooksDir(), Equals, fmt.Sprintf("%s/name/1/meta/hooks", dirs.SnapMountDir))
 	c.Check(info.DataDir(), Equals, "/var/snap/name/1")
 	c.Check(info.UserDataDir("/home/bob"), Equals, "/home/bob/snap/name/1")
-	c.Check(info.HomeDirBase("/home/bob"), Equals, "/home/bob/snap/name")
```

Comment:
Curious why you removed this. I think we could special case `/root` to return `/root/snap` here.

Reply:
We don't have `HomeDirBase()` anymore and we don't use this anywhere so I removed the test.